controllers/controllers/workloads/env/builder.go

```diff
@@ -138,9 +138,11 @@ func (b *ProcessEnvBuilder) buildPortEnv(ctx context.Context, cfApp *korifiv1alp
 	processPorts := ports.FromRoutes(cfRoutesForProcess.Items, cfApp.Name, cfProcess.Spec.ProcessType)
 	if len(processPorts) > 0 {
 		portString := strconv.FormatInt(int64(processPorts[0]), 10)
+		cfInstancePorts := fmt.Sprintf("[{\"internal\":%s},{\"internal\":%s}]", portString, portString)
```

Comment:
Why do we need `internal` twice? I guess we need `external` as well?

Reply:
very good question! So i basically copied what i saw that a productive CF instance gave me by default. I got there:

CF_INSTANCE_PORTS env is: [{"internal":8080,"external_tls_proxy":61157,"internal_tls_proxy":61001},{"internal":8080,"internal_tls_proxy":61443}]

so my assumption is, that the first {} holds ALL ports and the second {} holds all "internal" ports. since I didn't know what the tls_proxy ports are in the korifi environment, I left them out for now.

Reply:
Could it be that in the example above you have had two routes for the same app? Docs for that env var do not seem to mention anything about the number of json entries.

Reply:
no, there is just one route when I do "cf routes". Could the route be hidden somehow? i didn't specifically create one...

Reply:
I have spun a traditional cf deployment, pushed dora from CATS and got

```
vcap@a0213476-fa1a-43c1-5ec8-99af:~$ echo $CF_INSTANCE_PORTS  | jq .
[
  {
    "external": 61000,
    "internal": 8080,
    "external_tls_proxy": 61002,
    "internal_tls_proxy": 61001
  },
  {
    "external": 61000,
    "internal": 8080,
    "internal_tls_proxy": 61443
  },
  {
    "external": 61001,
    "internal": 2222,
    "external_tls_proxy": 61003,
    "internal_tls_proxy": 61002
  }
]

``` 

weird...

Reply:
Talked to a CF developer:
proposal: once external_tls_proxy and internal_tls_proxy gets added, we could simply add the internal only (external ones not exposed if envoy is active). For now, internal should be enough but there seem to be apps, that rely on the existance of all fields